cmake/configs/modes.cmake

```diff
@@ -222,3 +220,6 @@ else()
     string(REPLACE "-g" "" CMAKE_CXX_FLAGS_RELWITHDEBINFO ${CMAKE_CXX_FLAGS_RELWITHDEBINFO})
 endif()
 
+if(GNU_C_COMPILER OR GNU_CXX_COMPILER)
```

Comment:
Should we set `no-strict-aliasing` for clang as well to be consistent?

Reply:
From my checking it doesn't support it.